src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -89,6 +90,7 @@ def default_api_version(self):
         ResourceType.MGMT_RESOURCE_POLICY: '2017-06-01-preview',
         ResourceType.MGMT_RESOURCE_RESOURCES: '2017-05-10',
         ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS: '2016-06-01',
+        ResourceType.MGMT_AUTHORIZATION: '2018-01-01-preview',
```

Comment:
Replace by:
```python
ResourceType.MGMT_AUTHORIZATION: SDKProfile('2018-01-01-preview',
    'classic_administrators': '2015-06-01'
)
```
This says "use 2018-01-01-preview for everything, except classic_administrators use 2015-06-01"